pytest_pylint.py

```diff
@@ -81,9 +81,14 @@ def pytest_collect_file(path, parent):
             ignore_list = ignore_string.split(',')
     except (NoSectionError, NoOptionError):
         pass
+    msg_template = None
+    try:
+        msg_template = pylintrc.get('REPORTS', 'msg-template')
+    except (NoSectionError, NoOptionError):
```

Comment:
`NoSectionError`, `NoOptionError` will be raised in first `try` if there is no `[MASTER]` section or `ignore` option in it.
`NoSectionError`, `NoOptionError` will be raised in second `try` if there is no `[REPORTS]` section or `msg-template` option in it.
So they are actually working independently.